adal/src/main/java/com/microsoft/aad/adal/SSOStateSerializer.java

```diff
@@ -62,15 +62,18 @@
             .registerTypeAdapter(TokenCacheItem.class, new TokenCacheItemSerializationAdapater())
             .create();
 
-    private int getVersion() {
-        return version;
+    /**
+     * No args constructor for use in serialization for Gson to prevent usage of sun.misc.Unsafe
+     */
+    @SuppressWarnings("unused")
```

Comment:
@weijjia I might be able to make this private. I just wanted to be consistent. Some of the classes are "newed" up with the default public/package private constructor in the tests. IS this the only constructor you want me to make private?